turbo/app/snapshots_cmd.go

```diff
@@ -1396,11 +1396,16 @@ func doRetireCommand(cliCtx *cli.Context, dirs datadir.Dirs) error {
 		return err
 	}
 	blockReader, _ := br.IO()
-	from2, to2, ok := freezeblocks.CanRetire(to, blockReader.FrozenBlocks(), coresnaptype.Enums.Headers, nil)
+	frozenBlocks := blockReader.FrozenBlocks()
+	if br.ChainConfig.Bor != nil {
+		frozenBlocks = min(frozenBlocks, blockReader.FrozenBorBlocks())
+	}
+	maxAvailableBlock := to
```

Comment:
here is correct: it must be stage_senders progress `stages.GetStageProgress(tx, stages.Senders)`. in some other places used stage_snapshots progress used